doc/base/classes.xml

```diff
@@ -341,7 +494,12 @@
 			<argument index="0" name="var" type="Variant">
 			</argument>
 			<description>
-				Returns the length of the given Variant if applicable. It will return character count of a String, element count of an Array, etc.
+				Returns the length of the given Variant if applicable. It element count of an Array, size of a Dictionary, etc.
```

Comment:
>It element

Also I mention String specifically because syntax difference of String and Array length is the reason `len` is added. (#1960)

Reply:
I removed String because it doesn't actually work with String. Perhaps its a bug?

Reply:
Probably an oversight: https://github.com/godotengine/godot/commit/c6120e77a8c1c7e946569453973629499673d414#commitcomment-23530427

Reply:
oh yes probably. I might have time after dinner to fix and if so ill update the docs.

Reply:
I've made #11161. If it's merged, `len` will work with String.